nativescript-core/ui/core/view/view-common.ts

```diff
@@ -819,9 +819,9 @@ export abstract class ViewCommon extends ViewBase implements ViewDefinition {
         }
     }
 
-    public requestLayout(): void {
+    public requestLayout(calledFromChild?:boolean): void {
```

Comment:
Space is needed after the colon in parameter to match the tslint rules
```suggestion
    public requestLayout(calledFromChild?: boolean): void {
```